doc/src/dump_image.rst

```diff
@@ -49,7 +49,12 @@ Syntax
        *tri* = color tflag width
          color = *type*
          tflag = 1 for just triangle, 2 for just tri edges, 3 for both
-         width = numeric value for tringle edge width (distance units)
+         width = numeric value for triangle edge width (distance units)
+       *ellipsoid* = color eflag level width
+         color = *type*
+         eflag = 1 for triangles, 2 for wireframe, 3 for both
+         level = mesh refinement level, value between 1 (low resolution) and 6 (ultra high resolution)
+         width = diameter of wireframe edges (distance units) (ignored for triangles)
        *body* = color bflag1 bflag2
          color = *type*
          bflag1,bflag2 = 2 numeric flags to affect how bodies are drawn
```

Comment:
can a brief description be added here? something like:
bflag1 = diameter of body particle
bflag2 = 1 for triangles, 2 for wireframe, 3 for both